.github/workflows/pr-release.yaml

```diff
@@ -0,0 +1,101 @@
+name: Release to @pr-<number> tag on npm
+on:
+  workflow_dispatch:
+  pull_request:
+    types:
+      - closed
```

Comment:
Can we add a manual dispatch for deleting all `pr-*` versions? this would be good to clean things.

Reply:
let's make this same action @luizstacio mentioned also remove `next` tags @pedronauck

Reply:
✅